packages/common/source/utils/index.ts

```diff
@@ -22,6 +26,37 @@ export const extractIds = (items: ObjectWithID[] | string[]): string[] => {
     });
 };
 
+export const sendEmail = async (emailAddress, subject, text, html) => {
+    try {
+        const message = {
+            to: emailAddress,
+            from: { name: "Hypertool", email: "noreply@hypertool.io" },
```

Comment:
1. `sendEmail` should accept an object instead of so many parameters.
2. The from name and email addresses should be specified by the caller. Don't hard code it here.

Reply:
Ok